person.rb

```diff
@@ -20,6 +21,11 @@ def correct_name
     @name
   end
 
+  def rent_book(rental)
+    rentals.push(rental)
+    rental.person = self
+  end
```

Comment:
- [ ] Kindly make sure your has-many relationship between the person and the rented book is implemented correctly.

hint: 
```rb
def add_rental(book, date)
  Rental.new(date, book, self)
end
```